pages/linux/nemo.md

```diff
@@ -1,16 +1,24 @@
 # nemo
 
-> File manager and graphical shell for Cinnamon.
-> More information: <https://github.com/linuxmint/nemo>.
+> Manages files and directories in Cinnamon desktop environment.
```

Comment:
```suggestion
> Manage files and directories in Cinnamon desktop environment.
```

Should imperative mood be applied here as well?

Reply:
Not really according to our style guide: it [requires](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood) imperative mood just for descriptions now.